public/style.css

```diff
@@ -70,15 +90,27 @@ form div {
 }
 
 .bestSeller {
-  height: 2.5rem;
+  height: 1.2em;
   width: 40%;
-  font-size: 2rem;
+  font-size: 1em;
```

Comment:
How about just a little bigger on bigger screen? I tested the following. The text won't come out of the banner.
.bestSeller {
  height: 1.5em;
  width: 50%;
  font-size: 1.2em;